dev/com.ibm.ws.microprofile.faulttolerance.cdi/src/com/ibm/ws/microprofile/faulttolerance/cdi/AbstractFTEnablementConfig.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2017 IBM Corporation and others.
+ * Copyright (c) 2018 IBM Corporation and others.
```

Comment:
2017, 2018 ??

Reply:
It's a new file, so I put 2018. However, git has picked up that it's somewhat similar to the 1.0 version of FTGlobalConfig. We could argue that the code is therefore old and the file should have 2017, 2018 in the header.